### PR TITLE
mgr/dashboard: nfs-ganesha: controller API documentation

### PR DESCRIPTION
**Pull request by rjfd**
This PR adds the API documentation for the NFS-Ganesha controller.

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

**Comment by rjfd** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/controllers/nfsganesha.py [hunk 1]
```diff
@@ -17,6 +17,61 @@
...
+    'access_type': (str, 'Export access type'),
+    'squash': (str, 'Export squash policy'),
+    'security_label': (str, 'Security label'),
+    'protocols': ([int], 'List of protocol types'),
+    'transports': ([str], 'List of transport types'),
+    'fsal': ({
+        'name': (str, 'name of FSAL'),
+        'user_id': (str, 'CephX user id', True),
+        'filesystem': (str, 'CephFS filesystem ID', True),
+        'rgw_user_id': (str, 'RGW user id', True)
```
Comment (jtlayton) [inline src/pybind/mgr/dashboard/controllers/nfsganesha.py]: sec_label_xattr should be here too

**Comment by jtlayton** [comment]
retest this please

**Comment by rjfd** [comment]
@jtlayton done

**Pull request merged**